src/controller.ts

```diff
@@ -5,19 +5,36 @@ import picomatch from "picomatch";
 import * as vscode from "vscode";
 import { DisposableStore, MutableDisposable } from "./disposable";
 import { last } from "./iterable";
-import { getContainingItemsForFile, ICreateOpts, ItemType, testMetadata } from "./metadata";
+import { ICreateOpts, ItemType, getContainingItemsForFile, testMetadata } from "./metadata";
 import { IParsedNode, parseSource } from "./parsing";
 import { RunHandler, TestRunner } from "./runner";
 import { ISourceMapMaintainer, SourceMapStore } from "./source-map-store";
 
 const diagnosticCollection = vscode.languages.createDiagnosticCollection("nodejs-testing-dupes");
-const jsExtensions = ".{mjs,cjs,js}";
+
+function jsExtensions(extensions: string[]) {
+  let jsExtensions = "";
+
+  if (extensions == null || extensions.length == 0) {
```

Comment:
We should check for this before creating the controller, and just not create any controllers if no extensions are defined. This is possible if a user is editing their settings and has autosave turned on.

Reply:
I'll implement the check tomorrow..

Reply:
I have implemented the check. If the array is empty, the following will be displayed:

![image](https://github.com/connor4312/nodejs-testing/assets/67872991/f2fd2d61-c761-40eb-9946-d14846f42845)